### PR TITLE
Add a razzle example

### PR DESCRIPTION
**Pull request by marcneander**
Here's an example on how to use `@loadable` with `razzle`. It's based on the merging of https://github.com/smooth-code/loadable-components/pull/161

### CODE REVIEW

**Comment by neoziro** [comment]
I tried to run it but it does not work:

```
λ razzle → λ git razzle-example → yarn start                                                                                                     130 ↵
yarn run v1.12.3
$ razzle start
 WAIT  Compiling...

✔ success client compiled in 3s 167ms
✔ success server compiled in 426ms

/Users/neoziro/Projects/loadable-components/examples/razzle/build/server.js:2935
!(function webpackMissingModule() { var e = new Error("Cannot find module '../build/loadable-stats.json'"); e.code = 'MODULE_NOT_FOUND'; throw e; }());
                                            ^

Cannot find module '../build/loadable-stats.json'
```

**Comment by marcneander** [comment]
It's based on the merging of https://github.com/smooth-code/loadable-components/pull/161

**Comment by neoziro** [comment]
@marcneander it does not work even if I run build before, can you check it please?

File: examples/razzle/razzle.config.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+module.exports = {
+    modify: (config, { target }) => {
+        const appConfig = Object.assign({}, config);
+
+        if (target === 'web') {
+            const filename = path.resolve(__dirname, 'build/loadable-stats.json'); 
+
+            appConfig.plugins = [
+                ...appConfig.plugins,
+                new LoadableWebpackPlugin({ writeToDisk: true, filename })
```
Comment (willdurand) [inline examples/razzle/razzle.config.js]: Is this `writeToDisk` argument actually used?

Comment (neoziro) [reply examples/razzle/razzle.config.js]: See #161

**Pull request merged**